packages/eslint-config-loris/README.md

```diff
@@ -0,0 +1,31 @@
+# eslint-config-loris
+
+[ESLint](http://eslint.org/) [config](http://eslint.org/docs/developer-guide/shareable-configs) for Yandex Maps codestyle.
```

Comment:
Disagree, `loris` is just a name. We use this style guide in maps projects. Why we should rename it to abstract loris name?